### PR TITLE
Add pod cpu hog docs

### PR DESCRIPTION
**Pull request by ksatchit**
- Include pod cpu hog experiment docs
- Include new experiment docs into sidebar 

(fulfills the doc requirement for https://github.com/litmuschaos/litmus/pull/1069)

### CODE REVIEW

File: website/sidebars.json [hunk 1]
```diff
@@ -19,9 +19,12 @@
           "container-kill",
           "pod-network-latency",
           "pod-network-loss",
+	  "pod-network-corruption.md",
+	  "pod-cpu-hog",
           "disk-fill",
           "disk-loss",
-          "cpu-hog"
+          "cpu-hog",
+	  "drain-node"
```
Comment (chandankumar4) [inline website/sidebars.json]: Some indentation issue

**Review by ispeakc0de — DISMISSED** [review]

File: website/sidebars.json (hunk 1 above)
Comment (ksatchit) [reply website/sidebars.json]: sidebars is showing up correctly in local website / view file also shows the correct indentation

**Review by amitbhatt818 — DISMISSED** [review]
/LGTM

File: website/sidebars.json (hunk 1 above)
Comment (chandankumar4) [reply website/sidebars.json]: okay!!

File: website/sidebars.json [hunk 2]
```diff
@@ -19,9 +19,12 @@
           "container-kill",
           "pod-network-latency",
           "pod-network-loss",
+	  "pod-network-corruption.md",
```
Comment (chandankumar4) [inline website/sidebars.json]: is `.md` extension required?

Comment (ksatchit) [reply website/sidebars.json]: fixed!

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**